.rubocop_todo.yml

```diff
@@ -0,0 +1,21 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2024-03-29 17:05:57 UTC using RuboCop version 1.62.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 2
+# This cop supports unsafe autocorrection (--autocorrect-all).
+# Configuration parameters: AutoCorrect.
+Lint/UselessAssignment:
+  Exclude:
+    - 'app/controllers/users/omniauth_callbacks_controller.rb'
+
+# Offense count: 3
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, AllowedPatterns.
+# URISchemes: http, https
+Layout/LineLength:
```

Comment:
Increasing the line length for the entire project is broader than we want, but I looked at the rubocop errors with this TODO removed and they're not too bad to fix, so I'll make those changes and can push a commit to fix.

Reply:
I pushed a commit, so I'll mark this comment thread as resolved.